README.md

```diff
@@ -16,9 +16,9 @@
 
 ### Migrations
 * change models
-* `docker exec -it $(docker-compose ps -q django) bash -c "python3 manage.py makemigrations [app]"`
-* optional, if you want to inspect the corresponding SQL syntax: `docker exec -it $(docker-compose ps -q django) bash -c "python3 manage.py sqlmigrate [app] [number]"`
-* `docker exec -it $(docker-compose ps -q django) bash -c "python3 manage.py migrate"`
+* `docker exec -it $(docker-compose ps -q django) bash -c "integreat-cms makemigrations [app]"`
```

Comment:
The command `integreat-cms` doesn't work with docker exec, because `.bashrc` isn't sourced in this case (resulting in an `bash: integreat-cms: command not found` error). So either you get into the container shell first, end then execute `integreat-cms`, or you call docker exec with `.venv/bin/integreat-cms`.